src/components/buttons/button-table-edit.jsx

```diff
@@ -113,7 +120,16 @@ class ButtonTableEdit extends React.Component {
 	 */
 	_handleChange = (inputName, event) => {
 		const state = {};
-		state[inputName] = event.target.value;
+
+		if (
+			inputName === INPUT_NAMES.ROWS ||
+			(inputName === INPUT_NAMES.COLS &&
+				event.target.value < MINIMUM_GRID_VALUE)
```

Comment:
I don't understand this conditional. It says:

- If input is "rows", set value to 1.
- If input is "cols" and value is less than 1, set value to 1.

Maybe you the parentheses are in the wrong place and you meant:

- If input is "rows" and value is less than 1, set value to 1.
- If input is "cols" and value is less than 1, set value to 1.

ie.

```js
if (
  (inputName === INPUT_NAMES.COLS || inputName === INPUT_NAMES.ROWS) &&
  event.target.value < MINIMUM_GRID_VALUE
)
```

Having said that, a common pattern for that kind of thing is just to use `Math.min()`:

```js
if (inputName === INPUT_NAMES.COLS || inputName === INPUT_NAMES.ROWS) {
    state[inputName] = Math.min(event.target.value, MINIMUM_GRID_VALUE);
}
```

Reply:
```javascript